integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/CarbonGlobalDictionaryRDD.scala

```diff
@@ -483,12 +483,12 @@ class CarbonGlobalDictionaryGenerateRDD(
 }
 
 /**
- * Set column dictionry patition format
```

Comment:
Have you check it in the whole project? patition has 25+

Reply:
OK